pkg/utils/rbacevaluate.go

```diff
@@ -32,3 +33,107 @@ func TemporaryEvaluate(md metautils.NiceMD) error {
 	}
 	return nil
 }
+
+func GetAllowedAdminGroups(md metautils.NiceMD) string {
+	var groups string
+	for _, g := range strings.Split(md.Get("groups"), semicolon) {
+		if g == "AetherROCAdmin" {
+			return "AllGroups"
+		} else {
+			groups = groups + semicolon + g
+		}
+	}
+	return groups
+}
+
+func IsAllowToDoSet(allowedGroups, prefixString string, deleteElem []*gnmi.Path, update, replace []*gnmi.Update) bool {
```

Comment:
I don't think this is going to work at all - it's embedding too much in to onos-config that should be handled in OpenPolicyAgent - as it's done for Get. Please see how Get builds a JSON and sends it to OPA. Also see the REGO rules in the helm chart that drives OPA

Here in onos-config there should be no mention of `enterprise` - we have lots of different uses for onos-config where the target is store, city, fabric etc